client.go

```diff
@@ -26,6 +26,8 @@ type Tweets interface {
 	ConnectToStream(ctx context.Context, ch chan<- ConnectToStreamResponse, errCh chan<- error, opt ...*ConnectToStreamOption) *ConnectToStream
 	VolumeStreams(ctx context.Context, ch chan<- VolumeStreamsResponse, errCh chan<- error, opt ...*VolumeStreamsOption) *VolumeStreams
 	RetweetsLookup(ctx context.Context, tweetID string, opt ...*RetweetsLookupOption) (*RetweetsResponse, error)
+	PostRetweet(ctx context.Context, uid string, tid string) (*PostRetweetResponse, error)
+	UndoRetweet(ctx context.Context, id string, stid string) (*UndoRetweetResponse, error)
```

Comment:
I think some parameters (e.g. uid, tid) are not clearly.
This args are shown user. So, we must define arg name more meaningful.

Reply:
I have changed args name to
```
uid -> userID
tid -> tweetID
stid -> sourceTweetID
```

Reply:
I'll change #112 as well.